python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -59,6 +59,18 @@
     UnresolvedNamedLambdaVariable,
     CallFunction,
 )
+
+# import the hash functions
+from pyspark.sql.connect.functions._hash_funcs import (  # noqa: F401
+    crc32,
+    hash,
+    md5,
+    sha,
+    sha1,
+    sha2,
+    xxhash64,
```

Comment:
if we're going to do this, I think it's better to import with a star, and have `__all__` at `functions/__init__.py`, and rename `_hash_funcs.py` -> `hash_funcs.py` as the namespace will be hidden anyway.

Reply:
sounds good